### ISSUE TITLE
Install without having to build wheel in Python

### ISSUE THREAD

**Issue by cybergazer** [duckdb/duckdb#14455]
### What happens?

The python version of duckdb requires building wheel, which takes a large amount of cpu usage and time just to install it. It would be much better if it were just a precompiled binary, making it take less cpu usage and be less prone to errors.

### To Reproduce

.

### OS:

Debian

### DuckDB Version:

latest

### DuckDB Client:

python

### Full Name:

Cybergazer

### Affiliation:

Cybergazer

### What is the latest build you tested with? If possible, we recommend testing with the latest nightly build.

I have tested with a stable release

### Did you include all relevant data sets for reproducing the issue?

Yes

### Did you include all code required to reproduce the issue?

- [X] Yes, I have

### Did you include all relevant configuration (e.g., CPU architecture, Python version, Linux distribution) to reproduce the issue?

- [X] Yes, I have

**Comment by Tishj** [comment]
Can you fill out this field as it is definitely relevant to your issue?

```
Hardware:
No response
```

Thanks

**Comment by szarnyasg** [comment]
@cybergazer both x86 and arm binaries are available for Python 3.7+

```bash
$ docker run --rm -it debian:latest /bin/bash
```
```bash
$ apt update && apt install -y python3 pip
# this installs DuckDB in two seconds
$ time pip install duckdb --break-system packages #hack
```
